contracts/evoting/types/ballots.go

```diff
@@ -401,6 +407,7 @@ type Question interface {
 	GetMaxN() uint
 	GetMinN() uint
 	GetChoicesLength() int
+	GetString() string
```

Comment:
In this case `GetID() string` feels more appropriate.

Reply:
Then, you can rename `selectStr` to `selectID`, same for rank and text.

Reply:
Ok, same thing for the constants in the test file, right ?